gpytorch/lazy/lazy_tensor.py

```diff
@@ -1733,8 +1733,7 @@ def __mul__(self, other):
         return self.mul(other)
 
     def __radd__(self, other):
-        #TODO: see if this should be self.add
-        return self.mul(other)
```

Comment:
wut?

Reply:
This was fixed by [0e08be](https://github.com/cornellius-gp/gpytorch/commit/0e08bef757d6ce8055bbaa09fba919b45ff3c851). At the time, it seemed to be in master so I left it.